test/metabase/query_processor/parameters/temporal_units_test.clj

```diff
@@ -1,4 +1,4 @@
-(ns ^:mb/driver-tests metabase.driver.common.parameters.temporal-units-test
+(ns ^:mb/driver-tests metabase.query-processor.parameters.temporal-units-test
```

Comment:
Moved this namespace since there's no corresponding `src` namespace and it doesn't really have anything to do with drivers in particular any more than any other QP test namespace does. Actually it probably belongs in `test/metabase/query_processor_test` but I don't want to move it again